config/global.js

```diff
@@ -11,6 +11,7 @@ const {
 	REDIS_URI,
 	REQUEST_TIMEOUT = 15000, // 15 sec
 	NODE_ENV = 'development',
+	JWT_SHOW_TIMEOUT_WARNING_SECONDS = 3600, // 60 min
```

Comment:
please simplify this file to do not write all the constant names twice.
i'm sure you will find a nice solution for that.

Reply:
please write a seperate ticket, have to be on staging today. @tofixx you already discuss this with cedric!?